.github/workflows/dart_ci.yaml

```
@@ -0,0 +1,52 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test_consume_*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sdk: [ stable, beta, dev ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dart-lang/setup-dart@v1
+        with:
+          sdk: ${{ matrix.sdk }}
+      - name: Install dependencies
+        run: dart pub get
+      - name: Validate dependencies
+        run: dart run dependency_validator
+      - name: Check formatting
+        run: dart format --output=none --set-exit-if-changed .
+        if: ${{ matrix.sdk }} == 'stable'
```

I'm seeing the formatting run on all three of the sdks in the matrix... I think this might need to be 

```suggestion
        if: ${{ matrix.sdk == 'stable' }}
```